conbench/entities/commit.py

```diff
@@ -895,8 +895,9 @@ def _get_response_retry_guts(self, url) -> Optional[dict]:
             # Expect the value to always be int-convertible.
             reqquota = int(resp.headers["x-ratelimit-remaining"])
             metrics.GAUGE_GITHUB_HTTP_API_QUOTA_REMAINING.set(reqquota)
-            # Temporary workaround, see metrics._periodically_set_q_rem()
-            metrics.gauge_gh_api_rem_set["set"] = False
+            # Setting this to `True` stops a thread from periodically setting
+            # this to -1, see metrics.periodically_set_q_rem()
+            metrics.gauge_gh_api_rem_set["first_value_seen"] = True
```

Comment:
The bug was: this coordination property just stayed `False`. It was supposed to be updated here to `True`. I think part of the reason was an inexpressive name where I confused myself.